pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -31,6 +31,8 @@ const (
 	createInvitationPath  = "/create-invitation"
 	receiveInvtiationPath = "/receive-invitation"
 	acceptInvitationPath  = "/accept-invitation/{id}"
+	connections           = "/connections"
```

Comment:
I thought python agent had this path as /; likewise for the next line /{id}? Am I recalling incorrectly?

Reply:
No it is /connections in the python agent. I just verified.

Reply:
No, they have `/connections` and `/connections/{id}`

![image](https://user-images.githubusercontent.com/29631944/64639786-94895100-d3d6-11e9-8f99-9e098fa5a7a5.png)

Reply:
Which is "/", is it not?

Reply:
Ok - the misunderstanding is that the other strings are missing the prefix :).

Reply:
As discussed, we should also have the name "connections" as its own const so that we can easily update this name later.

Reply:
Fixed